### ISSUE TITLE
infinite loop in exception handler

### ISSUE THREAD

**Issue by craigh** [zikula/core#2142]
when there is a RouteNotFoundException the handler automatically sends the user to the route reload page, but if THAT is the route not found, then there is an infinite loop...

**Comment by craigh** [comment]
@cmfcmf - to attempt to fix this, I need to be able to do something like:

```
if (can't match route to `zikularoutesmodule_route_reload`) {
    reload routes for ZikulaRoutesModule;
}
```
proceed with redirect ...

how to do that?

**Comment by cmfcmf** [comment]
Like so: https://github.com/zikula/core/blob/1.4/src/system/ZikulaRoutesModule/Controller/RouteController.php#L213-L233. It could also be moved into an API function or something.

**Comment by Guite** [comment]
Probably fixed by #2183
